main/guides/orchestration/index.md

```diff
@@ -10,6 +10,10 @@ The Agoric Orchestration API sits on top of Agoric’s novel VM that provides th
 
 Agoric’s Orchestration APIs simplify controlling accounts on remote chains, moving assets, and using capabilities on any chain the API reaches.
 
+<br/>
+<img src="./assets/orch_arch.svg" width="125%" />
+<br/>
```

Comment:
Nice figure. I'd like to see it right after the 1st paragraph (ending "novel cross-chain-focused products.")

Reply:
Also, how do we maintain it? Is this .svg file the editable source form? What tool did you use to create it?

Reply:
@dckc I created the original image using latex in ipe tool, and yes this was SVG. 

As per offline comments by @toliaqat I have updated the image with a new version, and moved it to [how-orchestration-works](https://ms-add-orch-images.documentation-7tp.pages.dev/guides/orchestration/how-orch-works.html) page. This image use the original source image from [Agoric Orchestration Docs document](https://docs.google.com/document/d/1xniiU38GcCKn66q-YurGf_DCA5w_2a8SE6dh-1O1M4c/edit) and was created use Canva online editor for which SVG is not available.

Reply:
Canva? I found it in some Google slides. Here's an SVG export:

![orch-arch](https://github.com/user-attachments/assets/6d628948-69c3-49aa-bee3-5f11e8b4dd0c)

Reply:
@dckc can you update the text in "User Story" box as per the image mudassir uploaded, and reupload the SVG?

Reply:
I don't think svg vs png is a debate that should delay improving docs. We already have both kinds of images in the repo and they solve the problem at hand. Optimizing the format should come when we have learned enough after many iterations and have nothing else to improve.

Reply:
> ... We already have both kinds of images in the repo and they solve the problem at hand.

Yes; we use .png for screenshots and, until now, we used .svg for diagrams.

> Optimizing the format should come when we have learned enough after many iterations and have nothing else to improve.

So only after many iterations we should consider accessibility? That's not a position I'm prepared to support.

https://www.ohchr.org/en/instruments-mechanisms/instruments/convention-rights-persons-disabilities#article-9